### PR TITLE
Tweaked readme: recent features/screenshot

### PR DESCRIPTION
**Pull request by mmarcon**
* Mentioned indexes overview now available in tree view.
* Removed statement about the shell functionality having issues on Windows.
* Linked to `mongosh` product page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Launch the MongoDB Shell from the command palette to quickly connect to the same
 
 ![MongoDB Shell](resources/screenshots/shell-launcher.png)
 
-_The shell binary needs to be in your `$PATH`. This functionality has some issues on Windows, we'll release an update with a fix soon._
+_The shell binary needs to be in your `$PATH`. The shell binary can be either the `mongo` shell or the new [MongoDB Shell](https://www.mongodb.com/products/shell) (`mongosh`). In the extension's settings you can switch between the two._
```
Comment (alenakhineika) [inline README.md:36]: Probably `between these two` or `between two`?

Comment (mmarcon) [reply README.md:36]: I think "between the two" is grammatically correct but I am happy to ask a native speaker to double check. @Anemy?

Comment (Anemy) [reply README.md:36]: `between the two` sounds good to me. `between these two` would also be fine.

**Review by alenakhineika — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (alenakhineika) [reply README.md:36]: Thx Rhys!

**Pull request merged**